seed/tests/test_tax_lot_property.py

```diff
@@ -110,6 +111,43 @@ def test_csv_export(self):
         # last row should be blank
         self.assertEqual(data[52], '')
 
+    def test_csv_export_with_notes(self):
+        multi_line_note = self.property_view.notes.create(name='Manually Created', note_type=Note.NOTE, text='multi\nline\nnote')
+        single_line_note = self.property_view.notes.create(name='Manually Created', note_type=Note.NOTE, text='single line')
+
+        self.properties.append(self.property_view.id)
+
+        columns = []
+        for c in Column.retrieve_all(self.org.id, 'property', False):
+            columns.append(c['name'])
+
+        # call the API
+        url = reverse_lazy('api:v2.1:tax_lot_properties-export')
+        response = self.client.post(
+            url + '?{}={}&{}={}&{}={}'.format(
+                'organization_id', self.org.pk,
+                'cycle_id', self.cycle,
+                'inventory_type', 'properties'
+            ),
+            data=json.dumps({'columns': columns, 'export_type': 'csv'}),
+            content_type='application/json'
+        )
+
+        # parse the content as array
+        data = response.content.decode('utf-8').split('\r\n')
+        notes_string = (
+            multi_line_note.created.astimezone().strftime("%Y-%m-%d %I:%M:%S %p") + "\n" +
+            multi_line_note.text +
+            "\n----------\n" +
```

Comment:
On my machine, this causes the file to import into excel across mulitple lines

![image](https://user-images.githubusercontent.com/1907354/71262444-f3773400-22fc-11ea-9f07-acf97b5a153e.png)

Did this work for you when opening in Excel?

Reply:
I uploaded an export from my machine called "2propswithnotes.xlsx" into our shared drive:
https://drive.google.com/drive/folders/1d2xdSEVcwpO1TFaItXA_1r-9WDmd0v2W

Admittedly, I don't have Excel on my machine. But I do see the notes contained in one cell when opening in both Apple Numbers (🤢 I know) and Google Sheets.
<img width="822" alt="Screen Shot 2019-12-20 at 10 09 02 AM" src="https://user-images.githubusercontent.com/30608004/71274931-8326dd80-2311-11ea-8638-80c03196a9ac.png">
<img width="1328" alt="Screen Shot 2019-12-20 at 10 11 37 AM" src="https://user-images.githubusercontent.com/30608004/71274933-8326dd80-2311-11ea-9923-dee4bfecb037.png">